merge_data/healthcare_FR_finess.mapping.json

```diff
@@ -110,6 +109,29 @@
       "healthcare": "yes"
     }
   },
+  {
+    "categories": [
+	"603"
+    ],
+    "items": [
+      8400
+    ],
+    "missing_osm": false,
+    "classes": 20,
+    "level": 3,
+    "title": "Maison de santé",
+    "tags_select": {
+      "healthcare": "centre"
+    },
+    "tags_generate1": {
+      "healthcare": "centre"
+    },
+    "tags_generate2": {
+      "healthcare:speciality": "general",
```

Comment:
It `general` or "multi"?

Reply:
Ça veut dire qu'il y a au moins un médecin généraliste.

Je pensais mettre aussi les autres disciplines paramédicales :
`healthcare:speciality=general;paediatrics;psychotherapist;speech_therapist;midwife;nurse`

Le wiki n'est pas très clair à ce sujet et on retombe sur le problème récurrent de l'interprétation des tags à valeurs multiples 😞